mongodbatlas/resource_mongodbatlas_encryption_at_rest_test.go

```diff
@@ -15,10 +14,8 @@ import (
 )
 
 func TestAccResourceMongoDBAtlasEncryptionAtRest_basicAWS(t *testing.T) {
-	var encryptionAtRest = matlas.EncryptionAtRest{}
-
 	resourceName := "mongodbatlas_encryption_at_rest.test"
-	projectID := "5d0f1f73cf09a29120e173cf"
+	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
```

Comment:
Env var - That is much better I think than hard coded but from here forward we should be consistent (do one or the other as the standard- I think env var should be the preferred.)